.github/workflows/check_and_publish.yml

```diff
@@ -0,0 +1,58 @@
+name: Check and publish
```

Comment:
Hey there! I see these changes are already in main, so they shouldn't be in this PR.

Could you please rebase your entire branch onto the latest main? That way, the PR will only contain your new work.  🙏 

Thanks!